contracts/GovHub.sol

```diff
@@ -89,18 +88,21 @@ contract GovHub is System, IParamSubscriber {
     _;
   }
 
-  function init() external onlyNotInit {
+  constructor(Registry registry, uint256 votingPeriod_, uint256 executingPeriod_, bytes memory initialMembers_) System(registry) {
     proposalMaxOperations = PROPOSAL_MAX_OPERATIONS;
-    votingPeriod = VOTING_PERIOD;
-    executingPeriod = EXECUTING_PERIOD;
-    RLPDecode.RLPItem[] memory items = INIT_MEMBERS.toRLPItem().toList();
+    votingPeriod = votingPeriod_;
+    executingPeriod = executingPeriod_;
+    _loadInitialMembers(initialMembers_);
```

Comment:
Calling virtual functions from the constructor is usually discouraged. I have allowed myself to do so here because in _loadInitialMembers is only used to populate with initial members and in no place does it attempt to read contract's pre-initialized state vars